books/RayTracingTheNextWeek.html

```diff
@@ -1659,8 +1724,9 @@
 Image Texture Mapping
 ====================================================================================================
 
-We used the hitpoint p before to index a procedure solid texture like marble. We can also read in an
-image and use a 2D $(u,v)$ texture coordinate to index into the image.
+We used the hitpoint p before to compute $(u,v)$ surface coordinates to index a procedure solid
```

Comment:
“From the hit point $\mathbf{P}$, we compute the surface coordinates $(u,v)$. We then use these to index into our procedural solid texture (like marble). We can also ...”